apiserver/facades/client/controller/service.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package controller
```

Comment:
Thanks @ycliuhw, I believe we are stopping this practice of putting all the services in their own file for the facade. I believe someone started this practiced and we all just followed to keep consistency. @wallyworld & @manadart can you confirm?

Reply:
I'm all for declaring interfaces close to the point where they are used as a general rule. The practice of using an `interfaces.go` file or similar goes back a long way. For new code, we should try and do things according to our current preferred approach.

Reply:
We can land this and agree on our preferred approach and fix if needed. For example, the line count to define all the interfaces is large and bloats the file containing the method business logic. We could consider a separate file which has the interfaces and the facade struct only, thereby allowing the interfaces to be defined close to where they are used as attributes on the facade struct, but because the method logic is in a separate file, it's not bloated. This is similar to how we've separated out construction of the facade struct into a separate register.go file. We could even consider adding the above to this file.